larq/callbacks.py

```diff
@@ -24,19 +27,32 @@ class HyperparameterScheduler(tf.keras.callbacks.Callback):
     verbose: int. 0: quiet, 1: update messages.
     """
 
-    def __init__(self, schedule, hyperparameter, optimizer=None, verbose=0):
+    def __init__(
+        self,
+        schedule: Callable,
+        hyperparameter: str,
+        optimizer: Optional[keras.optimizers.Optimizer] = None,
+        verbose: Optional[int] = 0,
+    ):
```

Comment:
Thanks for the typehints. Unfortunately they don't show up nicely in our docs. I guess I need to refine our API docs generation. Though I think this shouldn't block this PR.

Reply:
@lgeiger looks like pydoc-markdown [doesn't have support for this yet](https://github.com/NiklasRosenstein/pydoc-markdown/issues/86).

Reply:
We could also try out https://github.com/keras-team/keras-autodoc which looks similar.

Reply:
For now we simply ignore the typehints in our rendered docs. I made some quick changes in to https://github.com/lgeiger/pydoc-markdown/commit/d6628ebd9f2ad5429c6898e63e5626873bdfc631